### ISSUE TITLE
Bintray storage disabled — switch to other storage for CI artifacts

### ISSUE THREAD

**Issue by Symbian9** [fontforge/fontforge#4730]
Bintray actually disabled its storage access and will be shutdown:

- https://bintray.com
> Thanks for supporting Bintray! This service has now been sunset, and to assist with migration to the JFrog Platform, paid accounts can login until July 4th. 

As a result FontForge artifacts NOT available anymore:

- ~~https://dl.bintray.com/fontforge/fontforge/~~

### REFERENCE

- https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#accessing-ci-build-archives

**Comment by jtanx** [comment]
Yes this is known but the documentation hasn't been updated. You can grab the latest from the artifacts on e.g. https://github.com/fontforge/fontforge/actions/runs/762939162

**Comment by ctrlcctrlv** [comment]
I'll write a documentation PR. Moving to fontforge/fontforge.github.io…
